fbw-a32nx/src/base/flybywire-aircraft-a320-neo/.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+    rules: {
+        'strict': 'off',
+        'no-unused-vars': 'warn',
+        '@typescript-eslint/no-unused-vars': 'warn',
+        'no-underscore-dangle': 'warn',
```

Comment:
I suggest removing this. A ton of the Asobo files use leading underscores heavily. This rule also makes getters and setters a bit awkward in general.